.gitignore

```diff
@@ -8,21 +8,13 @@ notebooks/.ipynb_checkpoints/
 # Metastore
 metastore_db/
 
-.gradle
```

Comment:
This the tmp dir that gradle use it for caching and user configs. We should ignore it.

Reply:
oh, didn't see it already there, nvm